sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -127,11 +127,11 @@ def sdc_pandas_dataframe_reduce_columns_df(df, func_name, params, ser_params):
     for key, value in ser_params.items():
         ser_par.append('{}={}'.format(key, value))
 
-    s_par = '{}'.format(', '.join(ser_par[:]))
+    s_par = '{}'.format(', '.join(ser_par))
```

Comment:
Why do you need to format the string? Why not do so:
`s_par = ', '.join(ser_par)`